packages/eslint-plugin/tests/rules/no-unnecessary-coercion.test.ts

```diff
@@ -0,0 +1,212 @@
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+
+import rule from '../../src/rules/no-unnecessary-coercion';
+import { getFixturesRootDir } from '../RuleTester';
+
+const rootDir = getFixturesRootDir();
+
+const ruleTester = new RuleTester({
+  languageOptions: {
+    parserOptions: {
+      tsconfigRootDir: rootDir,
+      project: './tsconfig.json',
+    },
+  },
+});
+
+const messageId = 'unnecessaryCoercion';
```

Comment:
(nit) Eh, i'd inline this

Reply:
changed this since it appears to only be done in a minority of test files, but strongly disagree for the record, as magic constants should be easy to change in one place